service/src/main/resources/quips/insultibot/insults.json

```diff
@@ -334,5 +334,117 @@
     "usageRules":[
       
     ]
+  },
+  {
+    "quipGroup":"CACTUS",
+    "text":"You must be a cactus.<break />  How else could you be so prickly?",
+    "ssml":"You must be a cactus.  How else could you be so prickly?",
+    "usageRules":[
+      
+    ]
+  },
+  {
+    "quipGroup":"ZOMBIE_OUTBREAK",
+    "text":"If there was a zombie outbreak, you'd be safe because they mainly seek out brains.",
```

Comment:
If there was a zombie outbreak you'd be safe, because they only eat people with brains.